### ISSUE TITLE
Request for new ontology [Occupation Ontology]

### ISSUE THREAD

**Issue by zhengj2007** [OBOFoundry/OBOFoundry.github.io#2428]
### Title

Occupation Ontology

### Short Description

Occupation Ontology The Occupation Ontology (OccO) is an ontology in the domain of human occupations.

### Description

OccO is an initial ontological presentation of information taken from a standardized occupation taxonomy, the US Standard Occupational Classification (SOC) as enhanced by the O*Net system. These systems have been developed by the following US Federal agencies:

- Department of Labor
- Bureau of Labor Statistics
- The O*NET Program, the United States' primary source of occupational information.

OccO reflects the data content of the O*Net-SOC 2019 taxonomy, which is based on the 2018 SOC, and it is developed by following OBO Foundry principles. The data used is from [O*Net data](https://www.onetonline.org/) by the U.S. Department of Labor, Employment and Training Administration (USDOL/EGA). It is used under the [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license.

### Identifier Space

OCCO

### License

CC-BY 4.0

### Domain

information

### Source Code Repository

https://github.com/Occupation-Ontology/OccO

### Homepage

https://github.com/Occupation-Ontology/OccO

### Issue Tracker

https://github.com/Occupation-Ontology/OccO/issues

### Contribution Guidelines

https://github.com/Occupation-Ontology/OccO/blob/main/CONTRIBUTING.md

### Ontology Download Link

https://raw.githubusercontent.com/Occupation-Ontology/OccO/main/occo.owl

### Contact Name

Jie Zheng

### Contact Email

zhengj2007@gmail.com

### Contact GitHub Username

zhengj2007

### Contact ORCID Identifier

0000-0002-2999-0103

### Formats

- [X] OWL RDF/XML (.owl)
- [ ] OBO (.obo)
- [ ] OBO Graph JSON (.json)

### Dependencies

- BFO
- OMO
- IAO
- NCBITaxon
- RO

### Related

- OMRSE

### Usages

```yaml
- user: Alabama government
  description: Alabama government employee related data integration and management
- user: https://www.hegroup.org
  description: Wikidata occupation data analysis
```

### Intended Use Cases and/or Related Projects

- Occupation classification data integration.
- Alabama government employee related data integration and management

### Data Sources

The primary data source used for current OccO ontology developer is the US SOC and O*Net occupation classification. The future, we will incorporate more data sources such as the ESCO data source. 

### Additional comments or remarks

References:
Foundational Development of an Occupation Ontology. [https://ceur-ws.org/Vol-3249/paper6-OSS.pdf](https://ceur-ws.org/Vol-3249/paper6-OSS.pdf)

Toward an Occupation Ontology, OccO. [https://icbo-conference.github.io/icbo2022/papers/ICBO-2022_paper_5061.pdf](https://icbo-conference.github.io/icbo2022/papers/ICBO-2022_paper_5061.pdf)

### OBO Foundry Pre-registration Checklist

- [X] I have read and understood [the registration process instructions](http://obofoundry.org/docs/Policy_for_OBO_namespace_and_associated_PURL_requests.html) and the [registration checklist](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md).
- [X] There is no other ontology in the OBO Foundry which would be an appropriate place for my terms. If there were, I have contacted the editors, and we decided in mutual agreement that a separate ontology is more appropriate.
- [X] My ontology has a specific release file with a version IRI and a `dc:license` annotation, serialised in RDF/XML.
- [X] My identifiers (classes and properties IRIs) are formatted according to the [OBO Foundry Identifier Policy](http://obofoundry.org/id-policy)
- [X] I understand that term definitions, while not mandatory, are key to understanding the intentions of a term especially when the ontology is used in curation. I made sure that a reasonable majority of terms in my ontology have definitions, using the [IAO:0000115](https://ontobee.org/ontology/IAO?iri=http://purl.obolibrary.org/obo/IAO_0000115) property.
- [X] For every term in my ontology, I checked whether another OBO Foundry ontology has one with the same meaning. If so, I re-used that term directly (not by cross-reference, by directly using the IRI).
- [X] For all relationship properties (Object and Data Property), I checked whether [the Relation Ontology (RO)](https://ontobee.org/ontology/catalog/RO?iri=http://www.w3.org/2002/07/owl%23ObjectProperty) includes an appropriate one. I understand that aligning with RO is an essential part of the overall alignment between OBO ontologies!
- [X] For the selection of appropriate annotation properties, I looked at [OMO](https://ontobee.org/ontology/catalog/OMO?iri=http://www.w3.org/2002/07/owl%23AnnotationProperty) first. I understand that aligning ontology metadata and term-level metadata is essential for cross-integration of OBO ontologies.
- [X] If I was not sure about the meaning of any of the checkboxes above, I have consulted with a member of the OBO Foundry for advice, e.g., through the [obo-discuss](https://groups.google.com/g/obo-discuss) Google Group.
- [X] The requested ID space does not conflict with another ID space found in other registries such as the Bioregistry and BioPortal, see [here](https://obofoundry.org/id-policy) for a complete list.

**Comment by hoganwr** [comment]
I'm going to weigh in carefully here.  As the developer of OMRSE, I have
been having behind the scenes negotiations with some (but not all)
proponents of OccO.  I would like to respectfully request that this request
be put on hold, and the Occo developers agree to it, until those
negotiations are complete.

This seems like a major orthogonality issue, with Occo have been started
with no consultation or discussions with OMRSE whatsoever.  We engaged OccO
at last year's (2022) ICBO, and had been having fruitful discussions.  OccO
told us they were not going to go with OBO foundry, but instead were going
with either IOF or another possibility (that I've forgotten now).

I overlooked the orthogonality issues on this basis.

Now that they've abruptly changed course and done with OBO, there is a
major issue, and we need to come to terms, which we had in process when I
was blindsided by this request.

**Comment by mbrochhausen** [comment]
I agree with Bill that this will create an orthogonality issue. It is
preferable for everyone that OccO and OMRSE finish their negotiations. From
what Bill says, it sounds like this ontology request might be premature.

Best,
Mathias

**Comment by yongqunh** [comment]
As I remember, we have been discussing and working with Bill Hogan, Matt Diller (Bill's PhD student), Bill Duncan, John Beverley, etc. on the occupation ontology (OccO) related collaborations since last year's ICBO 2022 conference. Sam Smith presented an OccO poster in ICBO 2022, and I introduced the work to Bill H., informed some possible overlapping with OMRSE, and initiated our discussions, negotiations, and collaborations. Matt has regularly attended our OccO working group meetings, and Bill has sometimes attended as well. Both Matt and Bill are OccO occupation ontology developers, and they are also co-authors of our recent OccO conference paper, which was presented by John Beverley in the the International Workshop on Ontologies for Social Services (OSS2023), July 2023, in Canada. 

The main overlapping exists in that OMRSE represents medically related social entities and OccO represents all occupations. OccO has ontologized all the occupations represented by the US Standard Occupational Classification (SOC), released by the US Bureau of Labor. Later we plan to integrate and incorporate occupation related terms from other occupational classification systems including the ESCO from European Union, etc. 

Both OMRSE and OccO follow the BFO upper level ontology. The high level terms and structural design have been well discussed since ICBO 2022 and have achieved agreement by our working group members including Matt and Bill H.

We have also discussed the submission of OccO to OBO Foundry (or another foundry such as IOF) for a long time. In the last few months we have basically achieved an agreement to submit it to OBO library. 

One major difference between OccO and OMRSE is that OccO represents all specific occupations under the branch of 'occupation holder' (a subclass of human), and OMRSE puts them under 'role'. So in OccO, we treat each occupation type (e.g. pharmacist) as a human being and OMRSE treats it as a job role. Both approaches appear to work well. Considering both ontologies following the same BFO design, the different representation emphases unde the same BFO can be the basis of the interoperabilty between the two ontologies.

**Comment by hoganwr** [comment]
First, I would like to confirm that much of what Oliver says is correct.

Where we disagree:
1. Matt Diller and I distinctly remember being the ones who made the
initial approach at ICBO 2022, when we saw the obvious encroachment on
OMRSE domain.  It's not a terribly important point: all agree that OccO was
well under way before OMRSE was aware of it.
2. We agreed to definitions of classes and their placement in certain
ontologies under the understanding OccO was not coming to OBO Foundry.
I've been clear about that all along with Matt Diller and relied on him to
communicate that at OccO meetings I could not attend.
3. The dispute is not very large and can be solved I believe by further
discussion and negotiation.
4. Therefore this request should be put on hold until that occurs.

Bill

**Comment by yongqunh** [comment]
Thanks, Bill, for your confirmation and clarification.  I would also like to clarify two things:

First, our work of developing an occupation focused ontology has lasted for a long time (about 10 years), initially with the name RoleO (Role Ontology), which was submitted to BioPortal: https://bioportal.bioontology.org/ontologies/ROLEO. I also presented a RoleO related poster presentation with a Wikidata use case demonstration in the BioCuration 2018 conference. The OccO is an updated version of RoleO with a new name and focus. Sam Smith, a primary developer of RoleO and later OccO, has been leading the technical development since the beginning.  

Second, I agree that initially we had no intention to submitting the OccO to OBO. We thought it might be sufficient to submit it to BioPortal. We discussed different options over the time. Recently Chris Mungall submitted an issue ticket to OccO GitHub:
https://github.com/Occupation-Ontology/OccO/issues/3 
To address the issue of making ontology IRI resolvable, we have followed Chris's suggestion, and Jie submitted the OccO to OBO for consideration.

Also, I agree with Bill that the dispute is not very large and can be solved. Bill, let's find time to discuss and negotiate on this. Thanks.

**Comment by pfabry** [comment]
OccO has been integrated in the [NOR dashboard](https://obofoundry.org/obo-nor.github.io/dashboard/index.html). The main issues are the following:

- Lack of definition for > 500 c;lasses: these are mainly classes that overlap with the Standard Occupational Classification: this is not a red flag.
- Duplicate alternative label, that is the same alternative label is used in different classes: this is not a red flag.
- Versioning red flag: the version IRI is well formed, so this is more a problem with the dashboard IMHO.

On the whole, the ontology passes the dashboard

**Comment by cthoyt** [comment]
> OccO has been integrated in the [NOR dashboard](https://obofoundry.org/obo-nor.github.io/dashboard/index.html). The main issues are the following:
> 
>     * Lack of definition for > 500 c;lasses: these are mainly classes that overlap with the Standard Occupational Classification: this is not a red flag.
> 
>     * Duplicate alternative label, that is the same alternative label is used in different classes: this is not a red flag.
> 
>     * Versioning red flag: the version IRI is well formed, so this is more a problem with the dashboard IMHO.
> 
> 
> On the whole, the ontology passes the dashboard

@pfabry it's more than the version IRI being well-formed, it actually has to point to the correct artifact

also, the alternative label problem is very worrysome, can we make this an explicit requirement that we don't add ontologies that have internal consistency issues?

**Comment by mbrochhausen** [comment]
I agree that the alternative label issue is concerning.

**Comment by hoganwr** [comment]
I'm curious to know what are the downsides of duplicates, is it specific to
the alternative label annotation property, and is there a better / approved
annotation property where duplicates are part of the use case or at least
permitted.

**Comment by pfabry** [comment]
@dosumis 
According to the [OBO operation Duty Rotation list](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg/edit#gid=715306719), the reviewer for this ontology is David Osumi-Sutherland. 
Could you please let us know your availability for this review?

**Comment by dosumis** [comment]
What's the timeline?

**Comment by pfabry** [comment]
> What's the timeline?

There is no deadline set, but a reasonable (few weeks) timeframe is expected.
However, this ontology is also the subject of ongoing discussions concerning orthogonality issues with OMRSE (cf. this issue). It seems relevant to wait for the conclusion of these discussions before going ahead with the review.

**Comment by yongqunh** [comment]
We (OccO team) have discussed with Bill Hogan @hoganwr at OMRSE, and have basically achieved an agreement on the harmonization.

**Comment by pfabry** [comment]
> We (OccO team) have discussed with Bill Hogan @hoganwr at OMRSE, and have basically achieved an agreement on the harmonization.

Great! Does it has a direct impact on the current release of OccO ? Should @dosumis wait for a new release before starting the review process ?

**Comment by hoganwr** [comment]
Yes, we created a new release of OMRSE which has terms OccO has agreed to
import and reuse.

**Comment by zhengj2007** [comment]
Based on the discussion, OccO will import the OMRSE and replace the OccO terms with OMRSE ones, see: https://github.com/Occupation-Ontology/OccO/issues/20

**Comment by matentzn** [comment]
@dosumis has been contacted again to provide this review as part of the general OFOC membership duties. "Remember you dont have to deal with this in Person. But you are responsible that this ontology gets reviewed by someone, by the next OFOC call (Oct 17th)."

**Comment by nlharris** [comment]
@dosumis did you find someone else to hand this off to? Please let us know.

**Comment by pfabry** [comment]
Hello @dosumis and @nlharris,

@balhoff has been assigned to review this ontology as he is the next in line in the [OBO operations Duty Rotation list](https://docs.google.com/spreadsheets/d/19GrEWVnpxjnrig0iYUOiUvsZ0JDbprMh1USnRb-SXtg).

In fact, @zhengj2007  was the next in line, but as she is directly involved in this ontology, I moved on to the next person.

**Comment by wdduncan** [comment]
@hoganwr will report back about whether he can do this.

**Comment by hoganwr** [comment]
I can do the review, but not in time for the next OFOC call.

**Comment by balhoff** [comment]
@zhengj2007 I also meant to suggest that you add [has ontology root term](http://purl.obolibrary.org/obo/IAO_0000700) annotations to your ontology.

**Comment by zhengj2007** [comment]
@balhoff Thanks for reviewing the OccO and for your valuable comments and suggestions. 

- We are planning to add the missing definitions, with a particular focus on high-level organizational terms.
- We will add [has ontology root term](http://purl.obolibrary.org/obo/IAO_0000700) annotations in OccO ontology.

**Comment by yongqunh** [comment]
That's great. Thanks, Jim @balhoff !

**Comment by zhengj2007** [comment]
Discussed on 2023-12-12 call, it is accepted.

**Comment by balhoff** [comment]
I've sent the acceptance email to @zhengj2007 and copied the OBO-discuss and OBO-operations mailing lists.

**Comment by zhengj2007** [comment]
@balhoff Thanks a lot!

What are the next steps? Do I need to create occo.md and OCCO.yml?

**Comment by balhoff** [comment]
Here is the email sent to the list:

@zhengj2007,

Thank you again for your ontology submission to the OBO Foundry. We are happy to inform you that your ontology (OccO) has been accepted following discussion in the OBO Operations Committee meeting, 2023-12-12. Before we can add it to the OBO ontology registry you need to complete the following steps.

Create a metadata record for your ontology to be included in the registry:
- Create a new file in https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology, called occo.md (there is an “Add file” button in the top right).
- Obtain the already curated metadata that relates to your ontology from https://github.com/OBOFoundry/obo-nor.github.io/blob/master/dashboard-config.yml (you'll need to scroll down to locate yours; it will have your prefix in the " - id:" field.)
- Create a pull request to add the metadata record. This pull request should include a link to this issue (the New Ontology Request issue).

Here is an example record for the PATO ontology: https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/pato.md?plain=1
Your metadata will be reviewed and merged by a member of the OBO Foundry Operations Committee. Permissible content for fields is being documented here: https://obofoundry.org/faq/permissible-metadata-content.html

To create a PURL registry entry for your ontology:
- Go to https://github.com/OBOFoundry/purl.obolibrary.org/tree/master/config, click “Add file” and add a file named occo.yml.
- Add the desired configuration.
- Make a pull request with a link to this issue See here for an example of a PURL yml file: https://github.com/OBOFoundry/purl.obolibrary.org/blob/master/config/pato.yml 

Best regards,
Jim
on behalf of the OBO Operations Committee

**Comment by yongqunh** [comment]
That's great news. Thanks, Jim and the OBO Operations Committee!
